temaakop/main.js

```diff
@@ -0,0 +1,63 @@
+const list = {}
+let toDo = false
+let inProgress = false
+let done = false
+
+let addTask = (task) =>  list[task] = '';
+
+
+let changeStatus = (task, status) => {
+    if (task in list) {
+        list[task] = status 
+    } else {
+        console.log('Use addTask')
+    }
+}
+
+let deleteTask = (task) => delete list[task];
+
+function showList() {
+
+  console.log(`\n To Do: \n`)
```

Comment:
Обрати внимание, по ТЗ для бонусной задачи пустые строки не нужны, а .log() сам по себе выводит данные на новой строке, так что \n здесь лишние
```suggestion
  console.log(` To Do: `)
```